{"metadata":{"modules":[{"name":"VigramSDK"}],"title":"LocalizedError Implementations","role":"collectionGroup"},"schemaVersion":{"major":0,"patch":0,"minor":3},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/failureReason","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/helpAnchor","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/recoverySuggestion"],"generated":true}],"kind":"article","hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/LocalizedError-Implementations"},"variants":[{"paths":["\/documentation\/vigramsdk\/authenticationerror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/errorDescription":{"title":"errorDescription","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","kind":"symbol","role":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror\/errordescription","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"abstract":[],"type":"topic"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/recoverySuggestion":{"title":"recoverySuggestion","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/recoverySuggestion","kind":"symbol","role":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror\/recoverysuggestion","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"abstract":[],"type":"topic"},"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"abstract":[],"title":"VigramSDK","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","type":"topic","url":"\/documentation\/vigramsdk","role":"collection","kind":"symbol","images":[{"type":"icon","identifier":"logo.png"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError":{"type":"topic","title":"AuthenticationError","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"AuthenticationError","kind":"identifier"}],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","abstract":[{"type":"text","text":"AuthenticationError contains custom failure events when connecting to Authentication process."}],"url":"\/documentation\/vigramsdk\/authenticationerror","kind":"symbol","role":"symbol","navigatorTitle":[{"text":"AuthenticationError","kind":"identifier"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/helpAnchor":{"title":"helpAnchor","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/helpAnchor","kind":"symbol","role":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror\/helpanchor","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"helpAnchor"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"abstract":[],"type":"topic"},"logo.png":{"identifier":"logo.png","alt":"Vigram logo.","type":"image","variants":[{"url":"\/images\/logo.png","traits":["1x","light"]}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/failureReason":{"abstract":[],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/failureReason","title":"failureReason","role":"symbol","type":"topic","url":"\/documentation\/vigramsdk\/authenticationerror\/failurereason","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"failureReason","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"kind":"symbol"}}}